### PR TITLE
PHP contact form

### PR DESCRIPTION
**Pull request by JessicaStrachan**
PHP contact form from this tutorial: http://www.freecontactform.com/email_form.php
Sends to email with message and email input, but doesn't pick up other inputs such as First Name, Surname and contact no. 

(Only hooked up with contact form on under Contact tab at the moment, wanted to get that one working first. Do you think it's possible to have two more forms working in this way?)

screen shot 2017-08-21 at 19 46 12

### CODE REVIEW

File: includes/custom-metaboxes/cmb2-events-fields.php [hunk 1]
```diff
@@ -40,7 +40,7 @@
 ]);
 
 $events_excerpt->add_field([
-  'desc' => __('Enter text to be displayed as excerpt on listing page card'),
+  'desc' => __('Enter text to be displayed as excerpt on listing page card' 'cmb2'),
```
Comment (harryfinn) [inline includes/custom-metaboxes/cmb2-events-fields.php:43]: think you need a comma here:

```
__('Enter text to be displayed as excerpt on listing page card', 'cmb2'),
```

File: contact.php [hunk 2]
```diff
@@ -2,6 +2,63 @@
...
+      my_contact_form_generate_response("error", $email_invalid);
+    else //email is valid
+    {
+        //validate presence of name and message
+        if(empty($name) || empty($message)){
+          my_contact_form_generate_response("error", $missing_content);
+        }
+        else //ready to go!
+        {
+            $sent = wp_mail($to, $subject, strip_tags($message), $headers);
```
Comment (harryfinn) [inline contact.php:54]: should this code be here? it looks as if this is trying to send an email too (if all the checks pass) but your form action is pointing to `send_form_email.php`?

File: contact-page.php [hunk 3]
```diff
@@ -0,0 +1,219 @@
...
+                    <input class="form__input form__input--no-right-space col-6" type="tel" id="number">
+                </div>
+            </div>
+            <label class="form__label" for="message">Your message</label>
+            <textarea class="form__input form__input--no-right-space col-12" maxlength="3000" rows="15" type="textarea" id="message"></textarea>
+            <input class="form__btn btn btn--secondary btn--submit" type="submit" name="submit">
+        </form>
+  	</div>
+  	<div class="js-form">
+        <form class="form grid js-research-form">
```
Comment (harryfinn) [inline contact-page.php:182]: does this need a form action on or does this one work by using the code at the top of the file OK?

File: contact-page.php [hunk 4]
```diff
@@ -0,0 +1,219 @@
+<?php
+/**
+* Template Name: Contact page
```
Comment (harryfinn) [inline contact-page.php:3]: looks like you have two of these files/templates plus the `send_form_email.php` - is this correct, do you need all 3?

File: contact-page.php (hunk 3 above)
Comment (JessicaStrachan) [reply contact-page.php:182]: The latter two media form and the research form are merely hardcoded at the moment. For now it is just the first form, Contact, under the contact tab. Just ignore these for now, but I would like to get them working in the same way if that's possible. @harryfinn

File: contact.php (hunk 2 above)
Comment (JessicaStrachan) [reply contact.php:54]: `send_form_email.php` is the code that captures the form and sends it to the email

**Pull request merged**